packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -284,100 +276,6 @@ export class MLSService extends TypedEventEmitter<Events> {
     return mlsResponse;
   }
 
-  public async getConferenceSubconversation(conversationId: QualifiedId): Promise<Subconversation> {
```

Comment:
This all was moved to `SubconversationService`. All the logic related with subconversaitons lives there now.